ui/pages/settings/index.js

```diff
@@ -1,25 +1,17 @@
 import Head from 'next/head'
 
 import Dashboard from '../../components/layouts/dashboard'
-import HeaderIcon from '../../components/header-icon'
 import Admin from './admin'
```

Comment:
We should move the `admin` component to `components/` or put it in this file if it isn't shared – since Next.js will create a page for it.